scaper/core.py

```diff
@@ -1828,9 +1898,21 @@ def generate(self, audio_path, jams_path, allow_repeated_label=True,
             (no reverberation) and 1 (maximum reverberation). Use None
             (default) to prevent the soundscape from going through the reverb
             module at all.
+        save_isolated_events : bool
+            If True, this will save the isolated event audio in a directory adjacent to the generated
+            mixture, or to the path defined by `isolated_events_path`. The sources sum 
+            up to the mixture. Isolated events can be found (by default) at `[audio_path]_events/foreground` 
+            and `[audio_path]_events/background'. Isolated event file names follow the pattern: 
+            `[role]_[label][count]`, where count is the index of the isolated event in 
+            self.fg_spec (this allows events of the same label to be added more than 
+            once to the soundscape without breaking things).
+        isolated_events_path : str
+            Like `audio_path`, this determines the path of the directory within which 
+            the isolated event audio will be saved.
```

Comment:
I believe this should just say:
"Path to folder for saving isolated events. If None, defaults to the parent folder of audio_path"